prototypes/templates.lua

```diff
@@ -154,4 +154,13 @@ if data.raw.item["se-space-transport-belt"] then
   }
 end
 
+if data.raw.item["se-deep-space-transport-belt-black"] then
+  templates["deep-space-"] = {
+    prerequisite_techs = {"se-deep-space-transport-belt"},
+    tint = {r=0,g=0,b=0},
+    base_underground_name = "se-deep-space-underground-belt-black",
+  }
+  templates["space-"]["next_prefix"] = "deep-space-"
```

Comment:
`next_prefix` determines what happens when you select the miniloader with an (unconfigured) upgrade planner.
Since I'm assuming the deep space belt has no successor, you should remove this entry, but possibly add an entry to the `space-` prefix indicating that `deep-space-` is the upgraded version.

Reply:
That is what I did here.
Since they are on individual `if data.raw.item[...] then` conditions for `se-space-transport-belt` and `se-deep-space-transport-belt-black`, I did not want the set the `next_prefix `for the  `space-` template within the `space-` area but i am extending `space-` with `next_prefix` from within the `deep-space-` template.
While `se-space-transport-belt` and `se-deep-space-transport-belt-black` obviously will only exist together because they come from the same mod, the way the code is structured, i think it makes the most sense like this.

Please review this piece again, you probably miss-read line 163 assuming i am modifying the `deep-space-` template here.